### PR TITLE
v.transects: make module Python3 compliant

### LINKED ISSUE #400: v.transects python error on GRASS 7.8.2
**Name of the addon**
v.transects

**Describe the bug**
```
Traceback (most recent call last):
  File "/home/giovanni/.grass7/addons/scripts/v.transects",
line 358, in <module>
    main()
  File "/home/giovanni/.grass7/addons/scripts/v.transects",
line 332, in main
    v = loadVector(vector)
  File "/home/giovanni/.grass7/addons/scripts/v.transects",
line 112, in loadVector
    vectorAscii = p.stdout.read().strip('\n').split('\n')
TypeError: a bytes-like object is required, not 'str'
```

**To Reproduce**
Just try run the tool.

### PR DESCRIPTION
**Pull request by ninsbl**
This should fix #400 It still runs with Python2.
@gioman could you please test?

### CODE REVIEW

**Comment by gioman** [comment]
@ninsbl works fine! thanks!

File: grass7/vector/v.transects/v.transects.py [hunk 1]
```diff
@@ -105,11 +105,11 @@ def loadVector(vector):
...
     v[i][j] = [ xij, yij ] ,i.e., jth vertex in ith line
     """
-    expVecCmmd = 'v.out.ascii format=standard input=' + vector
+    #expVecCmmd = grass.encode('v.out.ascii format=standard input={}'.format(vector))
 # JL    p = Popen(expVecCmmd, shell=True, stdin=PIPE, stdout=PIPE, stderr=STDOUT, close_fds=True)
-    p = Popen(expVecCmmd, shell=True, stdin=PIPE, stdout=PIPE,
-              stderr=STDOUT, close_fds=False)
-    vectorAscii = p.stdout.read().strip('\n').split('\n')
+    #p = Popen(expVecCmmd, shell=True, stdin=PIPE, stdout=PIPE,
+    #          stderr=STDOUT, close_fds=False)
```
Comment (petrasovaa) [inline grass7/vector/v.transects/v.transects.py:111]: I would remove this, that's why we use versioning system:)

File: grass7/vector/v.transects/v.transects.py [hunk 2]
```diff
@@ -105,11 +105,11 @@ def loadVector(vector):
...
+    vectorAscii = grass.read_command("v.out.ascii", format="standard", input=vector).strip('\n').split('\n')
```
Comment (petrasovaa) [inline grass7/vector/v.transects/v.transects.py:112]: strip().split() should work just fine here

**Review by petrasovaa — COMMENTED** [review]
I know it's late but I added some notes, more style issues.

**Pull request merged**